SourceX/miniwin/misc.cpp

```diff
@@ -395,11 +423,13 @@ BOOL UpdateWindow(HWND hWnd)
 
 BOOL ShowWindow(HWND hWnd, int nCmdShow)
 {
+#ifndef USE_SDL1

```

Comment:
```suggestion
#ifdef USE_SDL1
	DUMMY();
#else
```

Reply:
Added a stub for SDL_Show/Hide/RaiseWindow with DUMMY impl.